shells/webextension/popups/disabled.html

```diff
@@ -2,7 +2,7 @@
 <style>
   html, body {
     font-size: 14px;
-    min-width: 380px;
+    min-width: 410px;
```

Comment:
I assume this is intentionally a different size from the others (given it was already that way) but just pointing it out.

Reply:
Yes, it's a different message that's less wide.